packages/app/src/components/search/GithubDiscussionsSearchResultListItem.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { ReactNode } from 'react';
+import Box from '@material-ui/core/Box';
+import Chip from '@material-ui/core/Chip';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import { Link } from '@backstage/core-components';
+import { ResultHighlight } from '@backstage/plugin-search-common';
+import { HighlightedSearchResultText } from '@backstage/plugin-search-react';
+import { IndexableDocument } from '@backstage/plugin-search-common';
+import { EntityRefLink } from '@backstage/plugin-catalog-react';
+
+export interface GithubDiscussionsDocument extends IndexableDocument {
```

Comment:
Just curious about using IndexableDocument vs SearchDocument as per the documentation's comment about:
> IndexableDocument is only useful for backends working directly with documents being inserted or retrieved from search indexes. When dealing with documents in the frontend, use SearchDocument